include/z64.h

```diff
@@ -327,7 +327,8 @@ typedef struct {
     /* 0x13C */ void* roomVtx;
     /* 0x140 */ s16  unk_140;
     /* 0x144 */ Vec3f rot;
-} SkyboxContext; // size = 0x150
+    /* 0x150 */ char unk_14A[0xE];
+} SkyboxContext; // size = 0x15E
```

Comment:
The field name and padding seems slightly incorrect here? (the resulting size being 0x160 either way)
```suggestion
    /* 0x150 */ char unk_150[0x10];
} SkyboxContext; // size = 0x160
```